pkg/kubelet/dockertools/manager.go

```diff
@@ -579,7 +579,13 @@ func (dm *DockerManager) runContainer(
 		}
 	}
 	memoryLimit := container.Resources.Limits.Memory().Value()
-	cpuShares := milliCPUToShares(container.Resources.Limits.Cpu().MilliValue())
+	cpuRequest := container.Resources.Requests.Cpu()
+	var cpuShares int64
+	if cpuRequest.Amount != nil {
+		cpuShares = milliCPUToShares(cpuRequest.MilliValue())
+	} else {
+		cpuShares = milliCPUToShares(container.Resources.Limits.Cpu().MilliValue())
```

Comment:
Please add a comment that explains what milliCPUToShares does if Limits.Cpu().Amount == nil.

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubelet/dockertools/docker.go#L315